test/testfixture/make_functions.go

```diff
@@ -53,10 +75,9 @@ func makeWorkItemLinkCategories(fxt *TestFixture) error {
 	fxt.WorkItemLinkCategories = make([]*link.WorkItemLinkCategory, fxt.info[kindWorkItemLinkCategories].numInstances)
 	wilcRepo := link.NewWorkItemLinkCategoryRepository(fxt.db)
 	for i := range fxt.WorkItemLinkCategories {
-		desc := "some description"
 		fxt.WorkItemLinkCategories[i] = &link.WorkItemLinkCategory{
 			Name:        testsupport.CreateRandomValidTestName("link category "),
-			Description: &desc,
+			Description: ptr.String("some description " + getTestFileAndLine()),
```

Comment:
Why the "some description" part here? It's repeated below also.

Reply:
I was there before and I wanted to keep it just for that reason. I didn't want the descriptions to be all technical about files line lines essentially.